### ISSUE TITLE
Does the policies.json file work on Windows?

### ISSUE THREAD

**Issue by WesleyBranton** [mozilla/policy-templates#187]
Hi. This isn't really an issue. I help write the documentation on the support forum knowledge base. We have an article about the policies.json which only includes steps for Mac and Linux, but I see here that you can also using the policies.json file on Windows. However, I wasn't able to get it working on my installation of Firefox ESR.

Can someone verify that the policies.json file works on Windows before we add the Windows steps to knowledge base? Thanks.

I was able to get the Group Policy method to work on Windows, not the policies.json file though.

**Comment by mkaply** [comment]
The policies.json file should definitely work on Windows, but if group policy is enabled, it won't be read.

So you have to make sure you have no group policies enabled.

**Comment by WesleyBranton** [comment]
@mkaply I was able to get it working on a Windows 10 Home version. Now that you mention it, on my Windows 10 Pro that I was originally testing on, I had tested the group policies on there, but they have since been removed and I ran gpupdate. Still no luck on that system, but it works on my other system, so I'm happy enough to include that in the documentation on SUMO.

Speaking of which, we need to get you KB review privileges on there because no one has approved the many Enterprise revisions pending, including the article that you looked at and gave the go ahead for.
